AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -45,15 +46,17 @@ def get_pixelsize(path_pixelsize_file):
         return pixelsize
 
 
-def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size=None):
+def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size=None, ellipse = False):
     """
     Find each axon and compute axon-wise morphometric data, e.g., equivalent diameter, eccentricity, etc.
     If a mask of myelin is provided, also compute myelin-related metrics (myelin thickness, g-ratio, etc.).
     :param im_axon: Array: axon binary mask, output of axondeepseg
     :param path_folder: str: absolute path of folder containing pixel size file
     :param im_myelin: Array: myelin binary mask, output of axondeepseg
+    :param ellipse: Boolean: if axon is considered ellipse, set to True else if considered circle, set to False.
```

Comment:
a circle *is* an ellipse, so i would personally go with the inverse logic, e.g: `circle_approx={True, False}`

Reply:
That seems reasonable, I will make changes.

Reply:
Done in commit 7d85055